cmd/gateway-azure.go

```diff
@@ -181,6 +156,65 @@ func azureToObjectError(err error, params ...string) error {
 	return e
 }
 
+func mustGetAzureUploadID() string {
```

Comment:
Can we not use mustGetUUID() ?  if not can we write a comment on this new implementation?

Reply:
Added comment